cms/static/cms/js/require-config.js

```diff
@@ -249,6 +251,21 @@
                 exports: "Logger",
                 deps: ["coffee/src/ajax_prefix"]
             },
+            "modernizr": {
+                exports: "Modernizr"
+            },
+            "afontgarde": {
+                deps: ["jquery", "modernizr"],
+                exports: "AFontGarde"
+            },
+            "edxicons": {
```

Comment:
A shim shouldn't be necessary as ``edx-icons`` is implemented using RequireJS. I'm surprised that this works, in fact.